Gems/Atom/Asset/Shader/Code/Source/Editor/ShaderAssetBuilder.cpp

```diff
@@ -149,6 +271,10 @@ namespace AZ
                 response.m_createJobOutputs.push_back(jobDescriptor);
             }  // for all request.m_enabledPlatforms
 
+            const AZStd::sys_time_t createJobsEndStamp = AZStd::GetTimeNowMicroSecond();
+            const u64 createJobDurationMicros = createJobsEndStamp - shaderAssetBuildTimestamp;
```

Comment:
I suggest using AZ::Debug::Timer instead. This is already used ShaderAssetBuilder::ProcessJob.

Reply:
I considered it, but the code was already capturing shaderAssetBuildTimestamp. In this case found it more convenient to simply pair it with another call to GetTimeNowMicroSecond().

Reply:
One reason I suggest this is it might be _better_ to decouple from shaderAssetBuildTimestamp. We had a bug in ProcessJob where the build timestamp was incorrect because someone accidentally used the debug timestamp, not realizing the units didn't match, and it broke shader hot-reload. I worry about someone re-introducing a similar bug in the future. It's saver if we can separate our debug data from business data, but I won't harp on it.